### ISSUE TITLE
[Feature Request] Following Fluent UI style

### ISSUE THREAD

**Issue by liusong1111** [svelteuidev/svelteui#332]
### Possible implementation - describe how the feature can be implemented

I hope svelteui's style be more Fluent UI compliant(looks).

There are so many styles provided by so many UI Component Libraries in the world(even one for each). But nowadays material/fluent styles are generally accepted, we don't need customization from ground by throwing away some basic design of material/fluent, which give end-user surprise and inconvenience. 

Maybe someone still need some customization on them, then a good theme or a whole `unstyled`/`headless` component system may help, that's another topic.

Since svelteui looks more like fluent, and this repo seems abandoned:
https://github.com/Tropix126/fluent-svelte

It's a good chance to pick up the design following fluent ui style.

### Do you want to contribute this feature and create a pull request

No

**Comment by drewbitt** [comment]
I have no interest in mimicking Fluent UI, personally. Are you asking all libraries to switch to Fluent? Which ones and why?

**Comment by liusong1111** [comment]
Thanks for your reply.

I was rude, sorry.

You make svelteui compelete, good looking, and open, that's wonderful. 

To be concise, Fluent is not the only choice. I am just searching a good looking out-of-box UI Component framework for svelte, and I did think svelteui looks like Fluent :) Now I know it's by chance. 

I checked several UI frameworks, and found their styles are various. part of them I like, and part of them I dislike. 
That's options I am facing.

Thanks for your reply again.
